tests/test_base.py

```diff
@@ -127,7 +138,7 @@ def test_simple_task(self):
 
         Worker(self.tiger).run(once=True)
         self._ensure_queues(queued={"default": 0})
-        assert not self.conn.exists("t:task:%s" % task["id"])
+        assert not self.conn.exists(f"{REDIS_PREFIX}:{TASK}:%s" % task["id"])
```

Comment:
I would keep these literals unchanged in tests to make it clearer what these keys look like.